app/models/spree/admin/resources/translator.rb

```diff
@@ -0,0 +1,19 @@
+module Spree
+  module Admin
+    module Resources
+      module Translator
+        def with_default_translator
```

Comment:
If it's default, maybe it could be applied by the builder class by default? And only overridden if you need to use a custom translator (that's when you'd call `with_custom_translator`?

Reply:
Good idea 🙂.